src/scripts/scripts/zone/shadowmoon_valley/shadowmoon_valley.cpp

```diff
@@ -176,7 +176,7 @@ CreatureAI* GetAI_mob_azaloth(Creature *_creature)
 
 enum 
 {
-    SPELL_SHADOWBOLT = 9613,
+    SPELL_SHADOWBOLT_9613 = 9613,

```

Comment:
vs complained about it, but i guess its no problem to name two Spells SPELL_SHADOWBOLT in the same .cpp file if they are defined at the right place befor they are used? so does he take the new defined SPELL_SHADOWBOLT for each script that follows after that?